server/controllers/ClientDashboardController.js

```diff
@@ -23,19 +23,28 @@ async function postGig(req, res) {
 
 async function getUsername(req, res) {
   const user = req.session.userClient;
+  console.log(user.username);
   const clientUsername = await UserClient.find({ username: user.username });

```

Comment:
Use findById, could be the username in the database doesn't match the username in the session.